### ISSUE TITLE
Logging in takes user to the wrong screen

### ISSUE THREAD

**Issue by marykatefain** [HarvestHub/GardenHub#71]
Everytime I log in I get redirected to /accounts/profile and am returned a "not found' error. It should go to the home screen. When I delete the extra parts in the URL I get to home screen just fine and am actually logging in alright.

**Comment by alexgleason** [comment]
I'm not convinced this is real. `/accounts/profile` does not even exist in the urlconf:

```python
    path('', views.HomePageView.as_view(), name='home'),
    path('login/', LoginView.as_view(), name='login'),
    path('logout/', views.LogoutView.as_view(), name='logout'),
    path('admin/', admin.site.urls),
    # Account
    path('account/', views.AccountView.as_view(), name='account'),
    path('account/settings/', views.AccountSettingsView.as_view(), name='account-settings'),  # noqa
    path('account/delete/', views.DeleteAccountView.as_view(), name='account-delete'),  # noqa
    path('activate/<uuid:token>/', views.account_activate_view, name='account-activate'),  # noqa
    # Orders
    path('orders/', views.OrderListView.as_view(), name='order-list'),
    path('orders/new/', views.OrderCreateView.as_view(), name='order-create'),
    path('order/<int:pk>/', views.OrderDetailView.as_view(), name='order-detail'),  # noqa
    path('pick/<int:plotId>/', views.PickCreateView.as_view(), name='pick-create'),  # noqa
    # Plots
    path('plots/', views.PlotListView.as_view(), name='plot-list'),
    path('plot/<int:pk>/edit/', views.PlotUpdateView.as_view(), name='plot-update'),  # noqa
    # Gardens
    path('gardens/', views.GardenListView.as_view(), name='garden-list'),
    path('garden/<int:pk>/', views.GardenDetailView.as_view(), name='garden-detail'),  # noqa
    path('garden/<int:pk>/edit/', views.GardenUpdateView.as_view(), name='garden-update'),  # noqa
    # API
    path('_api/crops/<int:pk>/', views.ApiCrops.as_view()),
```

I'm almost certain your browser cached this as a redirect at some point. Try doing it in a different browser and see if it happens. I can't reproduce this issue on my browser.

**Comment by alexgleason** [comment]
Sorry, I'm wrong. I'm reproducing it on the staging server but not on my local development version.

I'm wondering if there's a cache elsewhere, like on the server itself (nginx)? I'll take a look at it. It's probably not an issue with the code.

**Comment by alexgleason** [comment]
I'm an idiot. :upside_down_face: I didn't even check the local dev version, I just assumed you were wrong. Sorry for being inadvertently condescending.

Check it out: https://docs.djangoproject.com/en/2.0/ref/settings/#login-redirect-url

`Default: '/accounts/profile/'`

Turns out when I switched to using Django's default LoginView I didn't consider where it would redirect to. Just need to change this to redirect to the homepage.
